crates/red_knot_python_semantic/resources/mdtest/type_properties/is_assignable_to.md

```diff
@@ -296,6 +305,28 @@ static_assert(is_assignable_to(Intersection[Any, Unrelated], Intersection[Any, P
 static_assert(is_assignable_to(Intersection[Any, Parent, Unrelated], Intersection[Any, Parent, Unrelated]))
 static_assert(is_assignable_to(Intersection[Unrelated, Any], Intersection[Unrelated, Not[Any]]))
 static_assert(is_assignable_to(Intersection[Literal[1], Any], Intersection[Unrelated, Not[Any]]))
+
+# TODO: No errors
+# error: [static-assert-error]
+static_assert(is_assignable_to(bool, Literal[False] | AlwaysTruthy))
+# error: [static-assert-error]
+static_assert(is_assignable_to(bool, Literal[True] | AlwaysFalsy))
+# error: [static-assert-error]
+static_assert(is_assignable_to(LiteralString, Literal[""] | AlwaysTruthy))
+static_assert(not is_assignable_to(Literal[True] | AlwaysFalsy, Literal[False] | AlwaysTruthy))
```

Comment:
These are duplicated from above. It seems they belong above (under "Union types" heading), not here under "Intersection types."
```suggestion
```